index.html

```diff
@@ -2,12 +2,172 @@
+    <meta name="keywords" content="HTML, CSS, JS, React" />
+    <meta name="author" content="Maxim Petrovsky" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Title</title>
   </head>
   <body>
   <h1>Моя первая страница</h1>
   <p>
     <a href="./about/index.html">Об авторе</a>
   </p>
+  <hr>
+  </body>
+  <body>
+  <img src="./images/frontend.jpg" alt="не знаю почему, но я орнул" />
+  <ol>
+    <li>
+      Знакомство с Web
+      <ul>
+        <li>Краткая история Web и что такое интернет.</li>
+        <li>Как устроен браузер и как он получает страницы по сети.</li>
+        <li>Что такое Git?</li>
+        <li>Создание первого репозитория.</li>
+        <li>О файле .gitignore.</li>
+      </ul>
+    </li>
+    <li>
+      HTML-вёрстка - первые шаги
+      <ul>
+        <li>Настройка рабочей среды.</li>
+        <li>Краткая история HTML.</li>
+        <li>Какие бывают теги и что такое атрибут.</li>
+        <li>Теги для работы с текстом.</li>
+        <li>Создание собственной страницы.</li>
+      </ul>
+    </li>
+    <li>
+      Погружение в HTML-разметку
+      <ul>
+        <li>Мета-теги.</li>
+        <li>Мнемоники.</li>
+        <li>Тег для работы с изображеними.</li>
+        <li>Теги для работы со списками.</li>
+        <li>Теги для работы с таблицами.</li>
+        <li>Теги для работы с формами.</li>
+        <li>Div-блоки.</li>
+      </ul>
+    </li>
+  </ol>
+  <table border="1" width="50%">
+    <caption>
+      <b>HTML tags for working with tables</b>
+    </caption>
+    <thead>
+    <tr>
+      <th>Tag</th>
+      <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>&lt;table&gt;</td>
+      <td>Defines a table</td>
+    </tr>
+    <tr>
+      <td>&lt;tr&gt;</td>
+      <td>Defines a row in a table</td>
+    </tr>
+    <tr>
+      <td>&lt;th&gt;</td>
+      <td rowspan="2">Defines a "header cell" / "cell" in a table</td>
+    </tr>
+    <tr>
+      <td>&lt;td&gt;</td>
+    </tr>
+    <tr>
+      <td colspan="2">Rarely used</td>
+    </tr>
+    <tr>
+      <td>&lt;caption&gt;</td>
+      <td>Defines a table caption</td>
+    </tr>
+    <tr>
+      <td>&lt;thead&gt;</td>
+      <td>Groups the header content in a table</td>
+    </tr>
+    <tr>
+      <td>&lt;tbody&gt;</td>
+      <td>Groups the body content in a table</td>
+    </tr>
+    <tr>
+      <td>&lt;tfoot&gt;</td>
+      <td>Groups the footer content in a table</td>
+    </tr>
+    </tbody>
+  </table>
+  <form action="./" method="get">
+    <fieldset>
+      <legend>My first form</legend>
+      <div>
+        <label for="name">Name</label><br />
+        <input required id="name" type="text" name="name" placeholder="Your name"/>
+      </div>
+      <div>
+        <label for="email">Email</label><br />
+        <input required
+               id="email"
+               type="email"
+               name="email"
+               placeholder="Your email"
+        />
```

Comment:
It is rather strange padding.
You may use 1 tab for every attribute.
```
<input
  required
  id="email"
  ...
/>
```
Or leave it in one row like row 106.